### PR TITLE
Now shading local stub variable if headers are being parsed

### PR DESCRIPTION
**Pull request by sypticus**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -26,7 +26,7 @@ allprojects {
 
 apply from: file('gradle/convention.gradle')
 apply from: file('gradle/release.gradle')
-apply from: file('gradle/maven.gradle')
+ // apply from: file('gradle/maven.gradle')
```
Comment (Xorlev) [inline build.gradle]: Unrelated change

**Comment by sypticus** [comment]
Fixed
